tests/server/server-construct-001.phpt

```diff
@@ -10,7 +8,13 @@ parse_url() tests must be reimplemented (PHPC-1177)
 <?php
 require_once __DIR__ . "/../utils/basic.inc";
 
-$parsed = parse_url(STANDALONE);
+$parsed = parse_url(URI);
```

Comment:
This is redundant in the case of a replica set. Ternary assignment would help here:

```php
$uri = is_replica_set(URI) ? get_primary_server(URI)->getInfo()['me'] : URI;
$parsed = parse_url($uri);
```